frontend/src/components/interview/components/history-section/components/interview-note-cards-list/interview-note-cards-list.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, InterviewNoteGetAllItemResponseDto } from 'common/types/types';
+
+import { InterviewNoteCard } from './components/components';
+import styles from './styles.module.scss';
+
+type Props = {
+  notes: InterviewNoteGetAllItemResponseDto[];
+};
+
+const InterviewNoteCardList: FC<Props> = ({ notes }) => {
+  return (
+    <div className={styles.notesBlock}>
+      {notes.map((note) => (
```

Comment:
```suggestion
      {notes.map(({ id, note, author, createdAt }) => (
```
Not to repeat `note.` multiple times